apps/system/js/bluetooth_transfer.js

```diff
@@ -644,15 +644,17 @@ var BluetoothTransfer = {
     var storage = navigator.getDeviceStorage(storageType);
     var getreq = storage.get(filePath);
```

Comment:
No. According to [1], elements passes into Promise.all([...]) are automatically wrapped with `Promise.resolve()` if it is not a promise.

According to [2], promise returned by `Promise.resolve()` only resolves when the `then()` of a "thenable" is called. So it's safe to pass the `getreq` into it directly and safely.

[1] https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/all
[2] https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/resolve